src/servers/ZoneServer2016/classes/character.ts

```diff
@@ -106,7 +106,7 @@ export class Character2016 extends Character {
         const { stamina, food, water, virus, health, bleeding } =
           client.character.resources;
         const { isRunning } = client.character;
-        if (isRunning) {
+        if (isRunning && (client.vehicle.mountedVehicle == "" || !client.vehicle.mountedVehicle)) {
```

Comment:
i think it's not the good way to do it, like isRunning whould be equal to false when driving a vehicle

Reply:
Setting isRunning to false when entering vehicle doesnt work 100% of the time, sometimes server received position packet from client after mounting so it gets set back to true